app/assets/javascripts/schools.js

```diff
@@ -0,0 +1,17 @@
+$(document).ready(function(){
+	$("form").on('click', '.remove_fields', function() {
+		$(this).prev('input[type=hidden]').val('1')
+		$(this).closest('fieldset').hide()
+		event.preventDefault()
+	});
+	$("form .add_fields").click(function(event){
+		time = new Date().getTime()
+		// g means global match
+		regexp = new RegExp($(this).data('id'), 'g')
+		//this line puts all the data before the link and replaces the ruby ids with time ids
+		$(this).before($(this).data('fields').replace(regexp, time))
+		event.preventDefault()
```

Comment:
In regular JS you should finish each line with `;`. JS compilers try to add it, but it's not safe to assume it will get added correctly to each line.